cli/azd/pkg/config/config.go

```diff
@@ -17,9 +17,8 @@ import (
 	"github.com/google/uuid"
 )
 
-//
 //nolint:lll
-var vaultPattern = regexp.MustCompile(
+var VaultPattern = regexp.MustCompile(
```

Comment:
What is the benefit of having `vault://id/id` references from the config/env to the vault?
Can't we just use the `vault:id` from the config to know there are secrets in config/env and directly use their key?